### PR TITLE
Exceptions

### PR DESCRIPTION
**Pull request by sundevour**
there's a new sourceHash with no source string at all, just a number
`860688654`, the Season 7 'Intrepid' armor sets

why??

this sets up an `exceptions` key so the config can basically contain manual source strings
these are added to the sourcesInfo early on so they are later checked with the rest of the config's strings

### CODE REVIEW

File: data/categories.json [hunk 1]
```diff
@@ -63,5 +63,8 @@
     "scourge":  [
       "Midnight Smith"
     ]
-  }
+  },
+  "exceptions": [
+      ["860688654", "Season 7 'Intrepid' armor sets"]
+  ]
```
Comment (delphiactual) [inline data/categories.json]: can we change this to an object?
```
exceptions: {
 "8606888654": "Season 7 'Intrepid' armor sets"
}
```
or does it work better working with an array of arrays?

**Comment by sundevour** [comment]
originally it was an object, but it's not referred to after being appended to the sourcesInfo
exceptions don't need to be looked up against, just pushed into the sourcesInfo object once as a pair. sourcesInfo was always the lookup table for categorization and annotation

**Comment by sundevour** [comment]
tldr either works. can arbitrarily switch if wanted
